components/landing/highlights.json

```diff
@@ -154,7 +154,7 @@
       "title": "",
       "text": "Blocking of QueerNet",
       "report": "https://ooni.io/post/ethiopia-report/#lgbti-websites",
-      "explore": "/search?input=queernet.org&probe_cc=ET",
+      "explore": "/search?domain=queernet.org&probe_cc=ET",
```

Comment:
This query is missing date ranges. cc @agrabeli

Reply:
This also results in zero results. I think it's missing the `www.`

Reply:
@hellais Queernet has both the queernet.org & www.queernet.org domains. You're right though to say that it's best to include the "www" version, since that's the version that's mostly been tested in ET.